### ISSUE TITLE
2008 temperatures

### ISSUE THREAD

**Issue by jrcunning** [hputnam/Moorea_Sym#7]
need to test that 2008 temps were not too different from 2006-2007, can use LTER data.

**Comment by hputnam** [comment]
Obtained MCR LTER Site 1 and 6 data
"A continuous time series of water temperature is measured with bottom-mounted thermistors at six sites around the shores of Moorea, on the fringing reef, backreef, and forereef. The forereef temperature is recorded at 10, 20, 30 and 40 m, the backreef at 1 m and the fringing reef at 1 and 5 m. SBE 39s are deployed to the forereef and backreef. Onset HOBOs are deployed on the fringing reef. Temperature data are processed and resampled to a 20 minute time step."


knb-lter-mcr.1035
Cite:
Leichter, J of Moorea Coral Reef LTER. 2015. MCR LTER: Coral Reef: Benthic Water Temperature, ongoing since 2005. knb-lter-mcr.1035.10 doi:10.6073/pasta/64651327973d5dfba8ef75f886f7d106

**Comment by hputnam** [comment]
Need to clean data after gaps and think about statistical tests as evidence supporting 2008-2009 data being the same patterns as 2006-2008. Goodness of fit test between distributions?

**Comment by hputnam** [comment]
@hputnam compare for all days for which data re avaialbel for site 2 hobo and site lter 1, is the temp distribution sig diff... if not, substitute lter 1 for tile site 2

**Comment by daniclaar** [comment]
I just cleaned up the data to remove any fluctuations due to deployment/retrieval. I also separated out the temperature pre-processing code into an R script that saves tempdata.RData so that we don't have to load all of the csvs each time. Temperature figures/visualizations are still in the data_exploration.Rmd. I re-ran the PCA, and the cleaning made a pretty big difference for the loadings (now the first 2 principal components explain 95% of the data). 

Note I haven't done any simultaneous trimming for the data sets (e.g. trimming to only dates where we have data for all loggers). After we test the Site 2 HOBO and LTER 1 for similarity, then we can move forward with that.
